drivers/input/input_pat912x.c

```diff
@@ -249,11 +325,25 @@ static int pat912x_pm_action(const struct device *dev,
 #endif
 
 #define PAT912X_INIT(n)								\
+	BUILD_ASSERT(IN_RANGE(DT_INST_PROP(n, res_x_cpi), 0, RES_MAX),		\
+		     "invalid res-x-cpy");					\
+	BUILD_ASSERT(IN_RANGE(DT_INST_PROP(n, res_y_cpi), 0, RES_MAX),		\
+		     "invalid res-y-cpy");					\
```

Comment:
it would be nice if a generous soul introduced min/max setting in our binding language. @fabiobaltieri maybe :-) ?

Reply:
You know what this project really needs? Interns.

Reply:
... or DT maintainers :eyes:

Reply:
> You know what this project really needs? Interns.

You mean like .. Summer of Code? :)